src/tests/intergration/repositories/test_answer_repository.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from sobesity.domain.entities.answer import AnswerEntity, AnswerFilterEnitity
+
+
+def test_get_list__empty_db__return_nothing(answer_repository):
+    assert answer_repository.get_list() == []
+
+
+def test_batch_create__get_all_rows(
+    answer_repository, answers, question_repository, questions, skill_repository, skills
+):
+    skill_repository.batch_create(skills)
+    created_skill = skill_repository.get_list()
+
+    for question in questions:
+        question.skill_id = created_skill[0].skill_id
+
+    question_repository.batch_create(questions)
+    created_questions = question_repository.get_list()
+
+    for answer in answers:
+        answer.question_id = created_questions[0].question_id
+
+    answer_repository.batch_create(answers)
+    created_answers = answer_repository.get_list()
+    created_answers_map = {answer.answer: answer for answer in created_answers}
+
+    assert len(created_answers) == len(answers)
+    for answer in answers:
+        assert answer.answer in created_answers_map
+
+
+def test_update__particular_rows_updated(
+    answer_repository, answers, question_repository, questions, skill_repository, skills
+):
+
+    skill_repository.batch_create(skills)
+    created_skill = skill_repository.get_list()
+
+    for question in questions:
+        question.skill_id = created_skill[0].skill_id
+
+    question_repository.batch_create(questions)
+    created_questions = question_repository.get_list()
+
+    for answer in answers:
+        answer.question_id = created_questions[0].question_id
+
+    answer_repository.batch_create(answers)
+    answer_before = answer_repository.get_list()[0]
+
+    to_set = answers[1]
+    where = AnswerFilterEnitity(answer_ids=[answer_before.answer_id])
+    updated_ids = answer_repository.update(to_set, where)
+
+    answer_after = answer_repository.get_list()[-1]
+
+    assert answer_before.answer_id == answer_after.answer_id == updated_ids[0]
+    assert answer_after.answer == to_set.answer
+
+
+def test_delete__particular_rows_deleted(
+    answer_repository, answers, question_repository, questions, skill_repository, skills
+):
+    skill_repository.batch_create(skills)
+    created_skill = skill_repository.get_list()
+
+    for question in questions:
+        question.skill_id = created_skill[0].skill_id
+
+    question_repository.batch_create(questions)
+    created_questions = question_repository.get_list()
+
+    for answer in answers:
+        answer.question_id = created_questions[0].question_id
```

Comment:
you can move all preparation of answers to fixture to not repeat the same code in each test